tools/install_demo_configuration.sh

```diff
@@ -410,9 +415,22 @@ OPENSEARCH_PLUGINS_DIR=`cd "$OPENSEARCH_PLUGINS_DIR" ; pwd`
 
 echo "### Success"
 echo "### Execute this script now on all your nodes and then start all nodes"
-#Generate securityadmin_demo.sh
+
+
+# Generate securityadmin_demo.sh
+# Run securityadmin_demo.sh only when creating new cluster
+if [ ! -z $UPGRADE ] ; then
+  echo "### This node is setup for upgrading existing cluster, skipping securityadmin.sh"
+  exit 0
+fi
+
+config_dir=securityconfig
+if [ ! -z $NEW_ODFE ]; then
```

Comment:
Given that this change will be in OpenSearch only, this flag doesn't seem to be necessary.